### ISSUE TITLE
keyError while getting liked songs

### ISSUE THREAD

**Issue by zak1388** [sigma67/ytmusicapi#113]
I got this error after calling `get_liked_songs()`.

`Traceback (most recent call last):
  File "ytmusic.py", line 5, in <module>
    liked_songs = ytmusic.get_playlist(playlistId='LM')
  File "/home/zak/.local/lib/python3.8/site-packages/ytmusicapi/mixins/playlists.py", line 100, in get_playlist
    playlist['tracks'].extend(parse_playlist_items(results['contents']))
KeyError: 'contents'`

Tested on Ubuntu 20.04, using python 3.8.5

**Comment by sigma67** [comment]
Have you liked any songs on this account? Do you see your liked songs when you go to https://music.youtube.com/playlist?list=LM?

**Comment by zak1388** [comment]
Yes, there's about 980 songs there

**Comment by sigma67** [comment]
The other library calls work for you? i.e. `get_library_playlists`

**Comment by zak1388** [comment]
Yep

**Comment by zak1388** [comment]
Nah it just returns a bunch of playlists called "test", and i cant access the playlist ids on the website either.

**Comment by sigma67** [comment]
That is very vague, please provide more specifics. This is impossible to debug at this point.

**Comment by zak1388** [comment]
`[{'title': 'test', 'playlistId': 'PLYpXmfKMEkF612pd0h_ICGCL_Ku9cAPE7', 'thumbnails': [{'url': 'https://lh3.googleusercontent.com/FoVQFdW6zBi3sNA_yZJSV3VTWmi0belhhFzleuEbn27utkirstj1woXHfWmWqkNyHla37ZFbk_F6jvVV=w226-h226-l90-rj', 'width': 226, 'height': 226}, {'url': 'https://lh3.googleusercontent.com/FoVQFdW6zBi3sNA_yZJSV3VTWmi0belhhFzleuEbn27utkirstj1woXHfWmWqkNyHla37ZFbk_F6jvVV=w544-h544-l90-rj', 'width': 544, 'height': 544}], 'count': '1'}, {'title': 'test', 'playlistId': 'PLYpXmfKMEkF4HdftoyUjGh1S9BZBptvPt', 'thumbnails': [{'url': 'https://i.ytimg.com/vi/bx1Bh8ZvH84/sddefault.jpg?sqp=-oaymwEWCJADEOEBIAQqCghqEJQEGHgg6AJIWg&rs=AMzJL3nTWReOwPrwHjPF5BprvzcbEpmMRw', 'width': 400, 'height': 225}, {'url': 'https://i.ytimg.com/vi/bx1Bh8ZvH84/hq720.jpg?sqp=-oaymwEXCKAGEMIDIAQqCwjVARCqCBh4INgESFo&rs=AMzJL3ldgnXFtqgQDnCMo_NfhGcXcqqbdw', 'width': 800, 'height': 450}], 'count': '1'}, {'title': 'test', 'playlistId': 'PLYpXmfKMEkF5CIUITqbqRu_gLs0nZAx1G', 'thumbnails': [{'url': 'https://i.ytimg.com/vi/bx1Bh8ZvH84/sddefault.jpg?sqp=-oaymwEWCJADEOEBIAQqCghqEJQEGHgg6AJIWg&rs=AMzJL3nTWReOwPrwHjPF5BprvzcbEpmMRw', 'width': 400, 'height': 225}, {'url': 'https://i.ytimg.com/vi/bx1Bh8ZvH84/hq720.jpg?sqp=-oaymwEXCKAGEMIDIAQqCwjVARCqCBh4INgESFo&rs=AMzJL3ldgnXFtqgQDnCMo_NfhGcXcqqbdw', 'width': 800, 'height': 450}], 'count': '1'}, {'title': 'test', 'playlistId': 'PLYpXmfKMEkF6uAkXZP_5X9fnUyfQTv5RR', 'thumbnails': [{'url': 'https://i.ytimg.com/vi/bx1Bh8ZvH84/sddefault.jpg?sqp=-oaymwEWCJADEOEBIAQqCghqEJQEGHgg6AJIWg&rs=AMzJL3nTWReOwPrwHjPF5BprvzcbEpmMRw', 'width': 400, 'height': 225}, {'url': 'https://i.ytimg.com/vi/bx1Bh8ZvH84/hq720.jpg?sqp=-oaymwEXCKAGEMIDIAQqCwjVARCqCBh4INgESFo&rs=AMzJL3ldgnXFtqgQDnCMo_NfhGcXcqqbdw', 'width': 800, 'height': 450}], 'count': '1'}]`
Basically it returns playlists, but these aren't any of them in my library, and i cant access them using their playlistIds

**Comment by sigma67** [comment]
If these aren't the right playlists you're probably using wrong credentials. Are you sure you used a request from your actual account for setup?

If you are, please provide the code you're using to setup the YTMusic instance.

**Comment by zak1388** [comment]
Alright, ill try setting the auth headers again

**Comment by zak1388** [comment]
Okay so i set the headers using `YTMusic.setup('headers_auth.json')` just like the instructions. That worked fine so i set the instance to a variable `ytm = YTMusic('headers_auth.json')` then called `print(ytm.get_library_playlists())` just to check the output. And I still got the same output from the earlier comment.

**Comment by sigma67** [comment]
Again, are you sure you're using the right account? Do you have brand accounts?

**Comment by zak1388** [comment]
Whoops, yep i think thats it, sorry about that. All good now, thanks man
